### ISSUE TITLE
Bug report: getting JWT token now has PyJWT issues

### ISSUE THREAD

**Issue by vineetparikh** [docusign/docusign-esign-python-client#152]
I'm currently using PyJWT 1.7.1 as packaged in the docusign-esign python client, and recently in our dev environment after a deploy we are getting the following issue when attempting to get a JWT token via `DSClient.get_token_jwt()`
```
NotImplementedError: Algorithm 'RS256' could not be found. Do you have cryptography installed? 
```
This is using the Docusign example, and the specific code this is failing at is
```
api_client.request_jwt_user_token(
                client_id=DS_JWT["ds_client_id"],
                user_id=DS_JWT["ds_impersonated_user_id"],
                oauth_host_name=DS_JWT["authorization_server"],
                private_key_bytes=private_key,
                expires_in=3600,
                scopes=use_scopes,
            )
```
Any thoughts as to what's going on? We aren't able to use Docusign in any form without fixing this.

**Comment by vineetparikh** [comment]
^ Sorry, should've been more specific. I had the `cryptography` package already installed, and I was getting this error. The version I was using was `cryptography 38.0.1`, which I verified based on `pip list`.

I proceeded to uninstall and reinstall `cryptography` via pip, and it looks like the version that was installed was version 2.8. I then reran this command and got the same output. I'm not sure what's going on at this point/why Docusign isn't picking up the pip installation of `cryptography`. For context, this is packaged in a Serverless framework Lambda function on AWS

**Comment by vineetparikh** [comment]
Yes, my application is already configured with an RSA keypair, so I don't think this is the issue. This was basically working for a while (and is still working in our production stages), but once I made a new deployment in a dev stage to add features, this error came up

**Comment by harsharahul** [comment]
@vineetparikh  Please reinstall requirements from the `requirements.txt` file if you have updated to the latest version. We updated the code to support the latest `pyjwt > 1.7` so the dependencies are having a miss-match.

**Comment by vineetparikh** [comment]
^ After attempting to reinstall from pip, it looks like docusign-esign's pip wheel is *not* updated to support pyjwt>1.7.1: could you please fix that?

**Comment by vineetparikh** [comment]
When I set the requirements file and attempt to install, I run into the issue described in https://github.com/docusign/docusign-esign-python-client/issues/153

**Comment by vineetparikh** [comment]
^ in my case I don't have a local setup of docusign-esign-python-client, I'm setting this up via pip and because I'm using Serverless, I can't actually make changes locally and see them on AWS. I would need this PR to be merged in
